pynsights/view.py

```diff
@@ -20,23 +19,27 @@
 lastCall = {}
 when = 0
 
-dump_filename = None
-index_output_filename = None
-index_input_filename = __file__.replace("view.py", "index.html")
```

Comment:
I had to refactor things a bit by removing the use of these globals. I know it's a cliché, but globals do make it really hard to refactor code.